bin/aurbuild

```diff
@@ -57,24 +57,24 @@ clean_cache() {
 }
 
 usage() {
-    plain "Usage: $argv0 [-c] -a <queue> -d <database> -p <pool> -r <root> [--] [<makepkg/makechrootpkg args>]"
+    plain "Usage: $argv0 [-c] [-s] -a <queue> -d <database> -p <pool> -r <root> [--] [<makepkg/makechrootpkg args>]"
 }
 
-
 trap 'exit' INT
 trap 'rm -rf "$tmp"' EXIT
 
 source /usr/share/makepkg/util.sh
 
 [[ -t 2 ]] && colorize
 
-while getopts :a:cd:p:r: OPT; do
+while getopts :a:cd:p:r:s OPT; do
     case $OPT in
         a) queue="$OPTARG"    ;;
         c) build=build_chroot ;;
         d) database="$OPTARG" ;;
         p) pool="$OPTARG"     ;;
         r) root="$OPTARG"     ;;
+        s) sign="--sign"      ;;
```

Comment:
The same option is used for both makepkg and repose to tell them to sign the package and the database respectively. A repose options array would allow specifying compression for the database, but you can't conveniently pass options to it by sticking them on the end of your command line, so it's a bit of a wash.